amplify/src/main/java/com/github/stkent/amplify/tracking/Amplify.java

```diff
@@ -72,27 +72,42 @@
     private String packageName;
     private String feedbackEmailAddress;
 
-    public static Amplify get(@NonNull final Context context) {
-        return get(context, new Logger());
+    public static Amplify initSharedInstance(@NonNull final Application app) {
```

Comment:
Yeah, I wanted to pass in the Application so I could automatically track an Activity, but then I didn't want to have to do that everywhere else. I plan on revisiting this stuff in #56 to see if it can be made clearer.